src/asammdf/blocks/v4_blocks.py

```diff
@@ -5465,7 +5469,7 @@ def time_stamp(self) -> datetime:
     def time_stamp(self, timestamp: datetime) -> None:
         if timestamp.tzinfo is None:
             self.time_flags = v4c.FLAG_HD_LOCAL_TIME
-            self.abs_time = int(timestamp.timestamp() * 10**9)
+            self.abs_time = int(timestamp.replace(tzinfo=timezone.utc).timestamp() * 10**9)
```

Comment:
In addition to making it more readable, the multiplier constant is used a lot so I suggest having a specific object with that value and multiplying by that value where necessary.

Idk, something like:

SECONDS_TO_NANOSSECONDS_MULTIPLIER = 10**9

self.abs_time = int(timestamp.replace(tzinfo=timezone.utc).timestamp() * SECONDS_TO_NANOSSECONDS_MULTIPLIER)